Extension/Context.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Extension;
+
+class Context
```

Comment:
do we need context to be an object? it seems to be a class that holds a string reference to another class. 
perhaps i missed the benefit benefit we gain by doing this?